src/Providers/DataGrid/Wijmo/Features/ContextMenu.ts

```diff
@@ -43,11 +43,9 @@ namespace Providers.DataGrid.Wijmo.Feature {
         private _addMenuItem(
             menuItem: OSFramework.DataGrid.Feature.Auxiliar.MenuItem
         ) {
-            //If already inserted to the Map return error message
+            //If already inserted to the Map return error message and exit the method
```

Comment:
An error message is not returned anymore, right?

Reply:
Correct @OS-giulianasilva